### ISSUE TITLE
[SF][DynamicPage][a11y]the visual page header should have a region role

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#2908]
The current documentation states that DynamicPage has a11yConfig to defines internally used a11y properties, however, it doesn't include the page header area(including page title and headerContent section).

**Is your feature request related to a problem? Please describe.**
 I'm frustrated when I use DynamicPage and a11y expert hopes page header region could be made a region for screen reader users.
![image](https://user-images.githubusercontent.com/10113427/166456525-c5e5015f-5b2d-4888-bf56-8f8dbede84c2.png)

**Describe the solution you'd like**
The circled area includes part of headerTitle and whole headerContent, I'm wondering how to make them read as a region. Maybe we can separate headerTitle from headerContent, and make each of them a region?

**Additional context**
Environment:
QACAND/TALCDPOMP/cgrant/pwd1234

https://qacand.hcm.ondemand.com/sf/OMP/Mentoring/MentorPreference?from_module=omp&p=mentee&programId=77355&appType=mentoringProgram&_s.crb=4aq%252b4U11%252frke1YIgTOier45VWCJZwC4cBVnz2TD7gpE%253d#/mentor-program/mentee/77355

**Comment by Lukas742** [comment]
Hi @tandan0813 

I'm not really sure what you want to achieve.
If you want to add a single `role` to a part of the `DynamicPageTitle` combined with the whole `DynamicPageHeader`? This is and will not be possible since both components are on the same level of the DOM tree and therefore their children cannot be combined to a single container. 
If you want to add a single `role` to the `DynamicPageTitle` combined with the `DynamicPageHeader` than this is also not possible, because again both components are mounted on the same level in the DOM and there's no wrapping component around these two except for the DynamicPage container itself.
If you want to add a `role` to each or only one of the components you can do so by adding the corresponding HTML Attribute as prop (in this case `role`).
